company.rb

```diff
@@ -0,0 +1,29 @@
+require './employees'
```

Comment:
Nice.  I like the extra file here for trying to get a feel for how to use what you've created.